src/base/searchengine.cpp

```diff
@@ -669,13 +670,20 @@ PluginVersion SearchEngine::getPluginVersion(QString filePath)
     if (!plugin.open(QIODevice::ReadOnly | QIODevice::Text))
         return {};
 
+    const PluginVersion invalidVersion;
+
     PluginVersion version;
     while (!plugin.atEnd()) {
         QByteArray line = plugin.readLine();
         if (line.startsWith("#VERSION: ")) {
             line = line.split(' ').last().trimmed();
-            version = PluginVersion::tryParse(line, {});
-            qDebug() << "plugin" << filePath << "version: " << version;
+            version = PluginVersion::tryParse(line, invalidVersion);
+            if (version == invalidVersion) {
+                LogMsg(tr("Search plugin '%1' contains invalid version string ('%2')")
+                    .arg(QFileInfo(filePath).fileName()).arg(QString::fromUtf8(line)), Log::MsgType::WARNING);
```

Comment:
Sorry, I don't understand. I wanted to print out a file name only, without path. How `Utils::Fs::toNativePath()` could help?